js/app.js

```diff
@@ -1,22 +1,31 @@
-var hobbies = [];
 
-function Hobby (opts) {
+var projects = [];
+
+function Project (opts) {
   this.title = opts.title;
   this.body = opts.body;
 }
 
-Hobby.prototype.toHtml = function() {
-  var $newHobby = $('article.template').clone();
-  $newHobby.find('h2').html(this.title);
-  $newHobby.find('.hobbies-body').html(this.body);
-  $newHobby.removeClass('template');
-  return $newHobby;
+Project.prototype.toHtml = function() {
+  // var $newProject = $('article.template').clone();
+  // $newProject.removeClass('template');
+  // $newProject.find('h2').html(this.title);
+  // $newProject.find('.projects-body').html(this.body);
+  // $newProject.removeClass('template');
+  // return $newProject;
+  var source = $('#project-template').html();
+  var template = Handlebars.compile(source);
+  return template(this);
 };
 
 myData.forEach(function(ele) {
-  hobbies.push(new Hobby(ele));
+  projects.push(new Project(ele));
+});
+
+projects.forEach(function(project) {
+  $('#projects').append(project.toHtml());
 });
 
-hobbies.forEach(function(hobby) {
-  $('#hobbies').append(hobby.toHtml());
+$('.icon-menu').click(function() {
```

Comment:
I like this, I did it using CSS and I think mine is a little buggy, so I might try it this way 👍